src/scales/scale.time.js

```diff
@@ -232,8 +232,9 @@ module.exports = function(Chart) {
 			var timeOpts = me.options.time;
 
 			// We store the data range as unix millisecond timestamps so dataMin and dataMax will always be integers.
-			var dataMin = Number.MAX_SAFE_INTEGER;
-			var dataMax = Number.MIN_SAFE_INTEGER;
+			// Integer constants are from the ES6 spec.
+			var dataMin = Number.MAX_SAFE_INTEGER || 9007199254740991;
```

Comment:
Only that it seems more future proof. It looks like Edge supports these, so in 'the future' the constants should be used everywhere and the hard coded values will never come into play. I've no strong opinion on this so let me know if you'd like it changed.